src/App.jsx

```diff
@@ -13,23 +11,9 @@ import DriverRoutes from './pages/dashboard/DriverRoutes';
 import DonateStatus from './pages/donation/DonateStatus';
 
 function App() {
-  const [files, setFiles] = useState([]);
-  const [images, setImages] = useState([]);
-
-  const onSubmit = async () => {
-    const url = await uploadImage(files[0]);
-    setImages(prev => [...prev, url]);
-  };
-
   return (
     <Router>
       <ChakraProvider>
-        <DropZone setFiles={setFiles} />
-        <Button onClick={onSubmit}>Upload File</Button>
-        {images.map(e => (
-          <Image key={e} src={e} />
-        ))}
```

Comment:
Bring the Dropzone + Button + Images back, I only meant for `DonationForm` to get deleted